roles/zuul/files/jobs/dib.yaml

```diff
@@ -0,0 +1,19 @@
+- job:
+    name: 'dib-build'
+    node: 'ubuntu-xenial'
+    builders:
+      shell: |
+        virtualenv $HOME/dib-build
+        . $HOME/dib-build/bin/activate
+        pip install diskimage-builder
+
+        git clone https://github.com/BonnyCI/hoist.git /opt/hoist
+        export ELEMENTS_PATH=/opt/hoist/roles/nodepool/files/etc/nodepool/elements/nodepool
+        disk-image-create -x -t qcow2 --checksum --no-tmpfs     \
+                          --qemu-img-options 'compat=0.10'      \
+                          -o /opt/nodepool/images/ubuntu-xenial \
+                          ubuntu-minimal vm
```

Comment:
@eggshell no, the elements path is where to find the listed elements, but this command only lists the two elements `ubuntu-minimal` and `vm`. I don't think there's a way to re-use the elements list from the `nodepool_diskimages` variable since it's scoped to a different host and all of these zuul jobs are files not templates, so for now the best thing might be to copy those elements into this command and add comments about keeping the two lists of elements in sync.